go.mod

```diff
@@ -1,85 +1,86 @@
 module github.com/KevinJoiner/crd-swagger
 
-go 1.20
+go 1.24.0
+
+toolchain go1.24.3
 
 require (
-	github.com/docker/docker v24.0.6+incompatible
-	github.com/docker/go-connections v0.4.0
-	github.com/rancher/wrangler/v2 v2.1.1-0.20230906224618-0a0c44968689
-	github.com/sirupsen/logrus v1.9.3
-	github.com/spf13/cobra v1.7.0
-	go.uber.org/zap v1.24.0
-	k8s.io/apiextensions-apiserver v0.28.0
-	k8s.io/apimachinery v0.28.0
-	k8s.io/client-go v0.28.0
-	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
+	github.com/docker/docker v28.1.1+incompatible
+	github.com/docker/go-connections v0.5.0
+	github.com/google/uuid v1.6.0
+	github.com/spf13/cobra v1.9.1
+	go.uber.org/zap v1.27.0
+	k8s.io/apiextensions-apiserver v0.32.0-alpha.0
+	k8s.io/apimachinery v0.32.0-alpha.0
+	k8s.io/client-go v0.32.0-alpha.0
+	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff
 )
 
+replace k8s.io/kube-openapi => github.com/pratikjagrut/kube-openapi v0.0.0-20250526080703-1864d16bcc00
```

Comment:
Would you mind copying the package / necessary functions inside crd-swagger instead of pointing to a fork?

We usually try to avoid forking repositories because they have high maintenance cost, we'd also have to create a repo under `rancher` org, etc. It will be simpler to just copy the three files.